addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -317,52 +334,19 @@ private synchronized void connect() {
         throw new IllegalArgumentException("");
     }
 
-    public synchronized void sendMessage(RFXComMessage msg) throws RFXComException {
-
-        ((RFXComBaseMessage) msg).seqNbr = getNextSeqNumber();
-        byte[] data = msg.decodeMessage();
-
-        logger.debug("Transmitting message '{}'", msg);
-        logger.trace("Transmitting data: {}", DatatypeConverter.printHexBinary(data));
-
-        setResponseMessage(null);
 
+    public void sendMessage(RFXComMessage msg) throws RFXComException {
         try {
-            connector.sendMessage(data);
+            RFXComBaseMessage baseMsg = (RFXComBaseMessage) msg;
+            baseMsg.rawMessage = baseMsg.decodeMessage();
```

Comment:
I switched back to generating the byte array every time. Getting decodeMessage to update rawMessage itself would touch every message class :-(